bin/migrate_to_single_table.py

```diff
@@ -180,6 +158,73 @@ def index_record_to_new_table(self, batch_size=1000):
                 )
             self.logger.info("Finished migrating :D")
 
+    @backoff.on_exception(
+        backoff.expo, Exception, max_tries=5, max_time=60, jitter=backoff.full_jitter
```

Comment:
I assume these values (especially `max_time` comes from some testing? It would nice if this script accepted that value as an argument but it's not necessary if you're certain this good after some tests. If this were code we expected to stick around, I'd argue for more config but I think it's okay